fido2/client.py

```diff
@@ -271,20 +273,48 @@ class WEBAUTHN_TYPE(six.text_type, Enum):
 _CTAP1_INFO = b"\xa2\x01\x81\x66\x55\x32\x46\x5f\x56\x32\x03\x50" + b"\0" * 16
 
 
+def running_windows_non_admin():
+    """
+    Determine if running on Windows with version >= 1903
+    and not as an administrator, which would require the use of
+    the built in WebAuthN API in Windows.
+    """
+    if platform.system().lower() == 'windows':
+        import sys
+        import ctypes
+        shell = ctypes.WinDLL('shell32')
+        build = sys.getwindowsversion().build
+
+        # Corresponds to the breaking 1903 update
+        # https://en.wikipedia.org/wiki/Windows_10_version_history#Rings
+        if not shell.IsUserAnAdmin() and build >= 18362:
+            return True
+
+    return False
+
+
 class Fido2Client(object):
     def __init__(self, device, origin, verify=verify_rp_id):
         self.ctap1_poll_delay = 0.25
         self.origin = origin
         self._verify = verify
         try:
-            self.ctap2 = CTAP2(device)
-            self.info = self.ctap2.get_info()
+            if running_windows_non_admin:
+                self.ctap2 = WinAPI()
+                self.info = self.ctap2.get_info()
+                self._do_make_credential = self._webauthn_make_credential
+                self._do_get_assertion = self._webauthn_get_assertion
```

Comment:
Last thing (I think), move `from .win_api import WinAPI` into here instead of the top of the file since importing it requires a dependency that is only installed on Windows and the import is only used on Windows when not run an admin. Else the top of file import will break macOS and Linux.

```suggestion
            if running_windows_non_admin:
                from .win_api import WinAPI
                self.ctap2 = WinAPI()
                self.info = self.ctap2.get_info()
                self._do_make_credential = self._webauthn_make_credential
                self._do_get_assertion = self._webauthn_get_assertion
```